arch/x86/vmx.c

```diff
@@ -1314,11 +1317,12 @@ int init_vmcs(struct vcpu *vcpu)
 	memcpy_s((void *) vcpu->arch_vcpu.vmcs, 4, &vmx_rev_id, 4);
 
 	/* Execute VMCLEAR on current VMCS */
-	status = exec_vmclear((void *)&vcpu->arch_vcpu.vmcs);
+	vmcs_pa = HVA2HPA((uint64_t)vcpu->arch_vcpu.vmcs);
```

Comment:
Why uint64_t here?  HVA2HPA takes va as parameter, it is good enough

Reply:
@dongyaozu , the function of exec_vmclear is define as: exec_vmclear(void *)
So one uint64_t variable is defined and assigned by HVA2HPA. Then exec_vmclear(&var) is called.

Reply:
But, it doesn't need the uint64_t inside (uint64_t)vcpu->arch_vcpu.vmcs)..

Reply:
@dongyaozu You are right. The uint64_t in HVA2HPA((uint64_t)vcpu->arch_vcpu.vmcs) is redundant. 
It seems that the one uint64_t is added in the macro of HVA2HPA.
So  Gen can remove it.

Reply:
I will remove the redundant uint64_t.